Task :sessionize:lib:testDebugUnitTest FAILED

I cloned the repository, opened the project with Android Studio and ran the command `./gradlew build`. The build failed with this message:

```
Task :sessionize:lib:testDebugUnitTest FAILED
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.robolectric.util.ReflectionHelpers$8 (file:/Users/alan/.gradle/caches/modules-2/files-2.1/org.robolectric/shadowapi/4.0/c917fd60b3b82353fbfdf36dc8008af489b8343e/shadowapi-4.0.jar) to method java.lang.ClassLoader.getPackage(java.lang.String)
WARNING: Please consider reporting this to the maintainers of org.robolectric.util.ReflectionHelpers$8
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

co.touchlab.sessionize.StaticFileLoaderTestJVM > testSpeakers FAILED
    java.lang.RuntimeException
        Caused by: java.lang.RuntimeException
            Caused by: java.lang.IllegalAccessException

co.touchlab.sessionize.StaticFileLoaderTestJVM > testAbout FAILED
    java.lang.RuntimeException
        Caused by: java.lang.RuntimeException
            Caused by: java.lang.IllegalAccessException
...
21 tests completed, 21 failed

FAILURE: Build failed with an exception.
```

--- a/sessionize/app/build.gradle
+++ b/sessionize/app/build.gradle
@@ -3,6 +3,8 @@ apply plugin: 'kotlin-multiplatform'
 apply plugin: 'kotlin-android-extensions'
 
 def firebaseEnabled = file("./google-services.json").exists()
+def releaseEnabled = file("./release.jks").exists()
+
 if (firebaseEnabled) {
     apply plugin: 'io.fabric'
 }
@@ -31,20 +33,24 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
 
     }
-    signingConfigs {
-        release {
-            storeFile file("release.jks")
-            keyAlias "key0"
-            storePassword "$releasePassword"
-            keyPassword "$releasePassword"
+    if(releaseEnabled) {
+        signingConfigs {
+            release {
+                storeFile file("release.jks")
+                keyAlias "key0"
+                storePassword "$releasePassword"
+                keyPassword "$releasePassword"
+            }
         }
     }
 
     buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-            signingConfig signingConfigs.release
+        if(releaseEnabled) {
+            release {
+                minifyEnabled false
+                proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+                signingConfig signingConfigs.release
+            }
         }
 
         //This is for MultiplatformSettings

--- a/sessionize/lib/src/androidTest/kotlin/co/touchlab/sessionize/TestOverridesJVM.kt
+++ b/sessionize/lib/src/androidTest/kotlin/co/touchlab/sessionize/TestOverridesJVM.kt
@@ -1,6 +1,7 @@
 package co.touchlab.sessionize
 
 import android.content.Context
+import android.util.Log
 import androidx.test.core.app.ApplicationProvider
 import androidx.test.ext.junit.runners.AndroidJUnit4
 import co.touchlab.sessionize.platform.AndroidAppContext
@@ -14,7 +15,9 @@ class StaticFileLoaderTestJVM : StaticFileLoaderTest() {
 
         ServiceRegistry.initLambdas({ name, type ->
             loadAsset("$name.$type")
-        }, { s: String -> Unit })
+        }, { s: String -> Unit }, {e:Throwable, message:String ->
+            Log.e("StaticFileLoaderTest", message, e)
+        })
 
         setUp()
 

--- a/sessionize/lib/src/commonTest/kotlin/co/touchlab/sessionize/NotificationTest.kt
+++ b/sessionize/lib/src/commonTest/kotlin/co/touchlab/sessionize/NotificationTest.kt
@@ -42,7 +42,8 @@ abstract class NotificationTest {
                 "schedule" -> SCHEDULE
                 else -> SCHEDULE
             }
-        }, {s: String -> Unit})
+        }, {s: String -> Unit},{e:Throwable, message:String ->
+        })
 
 //        AppContext.initAppContext()
 

